ts/Extensions/ParallelCoordinates.ts

```diff
@@ -691,8 +691,31 @@ namespace ParallelAxis {
                 }
             });
 
-            axis.dataMin = arrayMin(currentPoints);
-            axis.dataMax = arrayMax(currentPoints);
+            // Take into account range series points as well (#15752)
+            let min: number = NaN,
+                max: number = NaN;
+
+            currentPoints.forEach(function (point): void {
+                if (isArray(point)) {
+                    max = arrayMax(point);
+                    min = arrayMin(point);
```

Comment:
I guess you're here trying to find extremes of yAxis. What if you have more series? Only the last series' points will be really taken into consideration

Reply:
You've got a point!